Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Devices/GoogleVR.cs

```diff
@@ -0,0 +1,77 @@
+#if ENABLE_VR && !DISABLE_BUILTIN_INPUT_SYSTEM_GOOGLEVR
```

Comment:
@StayTalm 
So the `#if ENABLE_VR` part I'm not 100% sure about. My thinking here was that with the package dependency broken/inverted for now, `#if UNITY_INPUT_SYSTEM_ENABLE_XR` doesn't seem like the right check. Seemed to me that if someone flipped the VR checkbox in player settings, the XR stuff here should come live in the input system. Does that make sense?

Reply:
The individual `DISABLE_BUILTIN_INPUT_SYSTEM_XXX` would be for the packages to get rid of the built-in classes.